temporian/core/operators/select.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Select operator."""
+from typing import List, Union
+
+from temporian.core import operator_lib
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.operators.base import Operator
+from temporian.proto import core_pb2 as pb
+
+
+class SelectOperator(Operator):
+    """Select operator."""
+
+    def __init__(
+        self, input_event: Event, feature_names: Union[str, List[str]]
+    ):
+        super().__init__()
+
+        # inputs
+        self.add_input("input_event", input_event)
+
+        # outputs
+        output_features = [
```

Comment:
This is effectively changing the orders of the features.
What about having the output features in the same order as "feature_names"?

In addition, it would be interesting for this function to fails is "feature_names" does not exist in "input_event".